skyportal/handlers/api/gcn.py

```diff
@@ -20,12 +20,16 @@
     GcnNotice,
     GcnTag,
     Localization,
+    LocalizationTile,
 )
 from ...utils.gcn import get_dateobs, get_tags, get_skymap, get_contour
 
 log = make_log('api/gcn_event')
 
 
+Session = scoped_session(sessionmaker(bind=DBSession.session_factory.kw["bind"]))
+
+
 class GcnEventHandler(BaseHandler):
     @auth_or_token
     def post(self):
```

Comment:
Note: below it does not look as though it is checked whether `xml` is present in the payload or not, so may result in error.

Reply:
Also, raises a bare exception, instead of using `self.error`

Reply:
Curious how duplicate tags are handled in this handler.

Reply:
What do you suggest for the tag stuff?